manager/controllers/app/modules/moduleinterface.go

```diff
@@ -70,90 +69,116 @@ func (m *Selector) GetError() string {
 // AddModuleInstances creates module instances for the selected module and its dependencies
 func (m *Selector) AddModuleInstances(args *app.ModuleArguments, item DataInfo) []ModuleInstanceSpec {
 	instances := make([]ModuleInstanceSpec, 0)
+	geo := m.selectGeography(item)
 	// append moduleinstances to the list
 	instances = append(instances, ModuleInstanceSpec{
-		AssetID: item.AssetID,
-		Module:  m.GetModule(),
-		Args:    args,
+		AssetID:   item.AssetID,
+		Module:    m.GetModule(),
+		Args:      args,
+		Geography: geo,
```

Comment:
Is this meant to remain geo in the future or will it change to cluster ID?

Reply:
Not sure. A clusterID as the first step. Perhaps, to extend to a list of possible clusterIDs to make it possible to decide later on clusterID that is common to many modules. I would start with a single clusterID.